rust/src/lib/nm/nm_dbus/gen_conf/keyfile.rs

```diff
@@ -60,7 +60,7 @@ fn zvariant_value_to_string(
         }
         zvariant::Value::Array(a) => {
             let mut ret = String::new();
-            for item in a.get() {
+            for item in a.iter() {
```

Comment:
Is this change right? Is it doing the same thing? I assume that zbus-v1 chose a weird name for returning the iterator, but just to be sure.

Reply:
It is correct. The old zvariant 2.10 has `pub fn get(&self) -> &[Value<'a>]`  but changed to `pub fn get<V>(&'a self, idx: usize) -> Result<Option<V>>` in 5.1.

Our original use is expecting a iterator auto-generated from a slice. So changing it to iter() is correct.